docs/agreements/branch-and-release.md

```diff
@@ -56,6 +56,8 @@ The `develop` branch should always be in a deployable state.
    the second release of the day would be `2021-02-09-a`, third release would
    be `2021-02-09-b`, and so on.
 
+   If there is release of a new version of a product with documentation changes, also tag with this format: product-<prod_key>-<version>. See [Format for tags](https://docs.google.com/spreadsheets/d/14IsDQnviZipgtNAPSPzmfIqRP3g8yAwyUGia7idP2yU/edit?usp=sharing).
```

Comment:
Oh! Handy! We should probably put the format documentation in this repo, if possible.

Reply:
I added the tag format file to the agreements folder. Is that an appropriate location? I was having trouble verifying the updated link. Can you check it for me?

Reply:
[opinion] I personally think `product/${prod_key}/${version}` is better than `product-${prod_key}-${version}`. This will allow for certain tools to display things more neatly.

![Screen Shot 2021-09-15 at 9 50 58 AM](https://user-images.githubusercontent.com/607420/133475872-28de52d3-91de-4662-8ebb-616b5a01effb.png)

Reply:
Thanks for the comments. I have incorporated them.